risc0/zkvm/sdk/rust/tests/cc_end_to_end_test.rs

```diff
@@ -0,0 +1,38 @@
+#![cfg(feature = "risc_cc")]
```

Comment:
I think this is preventing CI from testing all this new code. Is there any harm in dropping the feature and just always including it?

Reply:
Its not ideal as the tests will be flaky for people who who don't have the toolchain installed, but its not a huge deal.